src/SearchButton.js

```diff
@@ -0,0 +1,13 @@
+import React from "react"; // imported react library
+
+/*extracted the search button from the search on a new component called search Button */
+const SearchButton = () => {
+  return (
+    // returned the button with JSX tags in the  search button component
+    <div>
```

Comment:
Why have you got this `div` here? It's not wrong, but I don't think it's necessary either...